src/gui/utils.cpp

```diff
@@ -164,7 +164,7 @@ void Utils::Gui::openFolderSelect(const Path &path)
     {
         if (SUCCEEDED(::CoInitializeEx(NULL, COINIT_APARTMENTTHREADED | COINIT_DISABLE_OLE1DDE)))
         {
-            PIDLIST_ABSOLUTE pidl = ::ILCreateFromPathW(reinterpret_cast<const wchar_t *>(path.toString().utf16()));
```

Comment:
Oops... This is my fault for producing a dangling pointer.
And it also seems that Qt developers still use `QString::utf16()` reinterpreted as `wchar_t` array.